djstripe/models/core.py

```diff
@@ -1218,6 +1239,284 @@ def is_valid_object(cls, data):
 File = FileUpload
 
 
+class PaymentIntent(StripeModel):
+	"""
+	Stripe documentation: https://stripe.com/docs/api#payment_intents
+	"""
+
+	stripe_class = stripe.PaymentIntent
+	stripe_dashboard_item_name = "payment intents"
+
+	amount = StripeQuantumCurrencyAmountField(
```

Comment:
@kavdev is StripeQuantumCurrencyAmountField the right type to use here?  (as opposed to StripeDecimalCurrencyAmountField).

See #247